File: JAVA/Daily Flash Codes/Week1/Day5_Jan17/code_Java/prog5.java


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		
		char ch;
		System.out.println("Enter the charcter: ");
		ch = (char)in.read();	//accpet character
		
		System.out.printf("ASCII of %c  : %d\n", ch,(int)ch );
	}


}

File: C/Daily Flash/Week1/Day5_Jan17/code_C/prog5.c
/*
 * Program 5: Write a program to ASCII values of input character.
Input: A
Output: ASCII Value of A is 65
*
*/

#include <stdio.h>

void main() {
	
	char c;
	printf("Enter one character: ");
	scanf("%c", &c);
	printf("ASCII value of %c : %d\n", c, c);
}
